lib/travis/build/appliances/services.rb

```diff
@@ -24,6 +24,15 @@ def apply
               end
             end
 
+            if services.delete('mysql')
+              sh.if "$(grep -E '^socket = /run/mysql-5.6' #{HOME_DIR}/.my.cnf)" do
+                sh.cmd "sudo service mysql-5.6 start", assert: false, echo: true, timing: true
```

Comment:
The situation is worse than I thought: https://travis-ci.org/meatballhat/cloaked-octo-tyrion/builds/180440706

I'm going to work on cleaning up the various mysql service bits, including aliasing `mysql-5.6` => `mysql`, although I'm pretty sure that upstart is going to have opinions about this.